packages/vitest-pool-workers/src/worker/lib/cloudflare/test-runner.ts

```diff
@@ -209,15 +209,6 @@ export default class WorkersTestRunner extends VitestTestRunner {
 		initialState.current = this.state.current;
 	}
 
-	async onBeforeRunFiles() {
```

Comment:
This has been removed in Vitest v3

Reply:
It is a bit strange that `onBeforeRunFiles` is removed while we are still using `onAfterRunFiles` 🤔 

Any chance it is a different issue? I can also find `onBeforeRunFiles` on vitest docs:
https://github.com/vitest-dev/vitest/blob/9acc13f367988ff5711f51d8b360a7f6d6602d2c/docs/advanced/runner.md?plain=1#L69-L76

Reply:
It's still in the docs, but it's been removed from the actual code

Reply:
https://github.com/vitest-dev/vitest/blob/9acc13f367988ff5711f51d8b360a7f6d6602d2c/packages/vitest/src/runtime/runners/test.ts